### ISSUE TITLE
Out of Memory issue

### ISSUE THREAD

**Issue by chirag-qs** [SceneView/sceneview-android#33]
I have tried sample app with my own customised .gltf 3D Model
but it is causing  java.lang.OutOfMemoryError: Failed to allocate a 134217744 byte allocation with 25165824 free bytes and 124MB until OOM, target footprint 95769496, growth limit 201326592 issue while rendering it. 
is it due to large size of 3D Model ? FYI my model size is 69 MB.

**Comment by ThomasGorisse** [comment]
Yes, it's due to your model size. 69 MB is big for a mobile usage.
There are a lot of ways you can reduce its size within blender. First thing you want to check is the texture size.
I handled to use gltfs upped to 25Mo but higher size may not be usable on low memory devices.
